cpp/include/cudf/types.hpp

```diff
@@ -110,8 +110,11 @@ enum type_id {
   FLOAT64,    ///< 8 byte floating point
   BOOL8,      ///< Boolean using one byte per value, 0 == false, else true
   DATE32,     ///< days since Unix Epoch in int32
-  TIMESTAMP,  ///< duration of specified resolution since Unix Epoch in int64
-  CATEGORY,   ///< Categorical/Dictionary type
+  TIMESTAMP_SECONDS,  ///< duration of seconds since Unix Epoch in int64
```

Comment:
Do we want to rename `DATE32` to `TIMESTAMP_DAYS` for consistency?

Reply:
Yes I think that's worth doing.

Reply:
Done in https://github.com/rapidsai/cudf/pull/2918/commits/9cd64c990b3ce0e694b61090cf94177ccf9530e8